sentences/de/_common.yaml

```diff
@@ -33,6 +33,60 @@ lists:
       type: "percentage"
       from: 0
       to: 100
+  cover_states:
+    values:
+      - in: "(offen|auf|oben)"
+        out: "open"
+      - in: "(geschlossen|zu|unten)"
+        out: "closed"
+      - in: "[am] (öffnen|öffnet)"
```

Comment:
as "am öffnet" does not make sense, I would suggest `(am öffnen|öffnet)` or `([am] öffnen|öffnet)`

Reply:
good point :+1: